LoopUI/Views/CGMStatusHUDView.swift

```diff
@@ -83,96 +101,27 @@ public final class CGMStatusHUDView: DeviceStatusHUDView, NibLoadable {
                                    at glucoseStartDate: Date,
                                    unit: HKUnit,
                                    staleGlucoseAge: TimeInterval,
-                                   sensor: SensorDisplayable?)
+                                   sensor: SensorDisplayable?,
+                                   statusHighlight: DeviceStatusHighlight?)
     {
-        // TODO refactor this function with LOOP-1293. Suggestion is to make a view model. Need to check with design about the display of stale glucose values.
-        var accessibilityStrings = [String]()
+        viewModel.setGlucoseQuantity(glucoseQuantity,
+                                     at: glucoseStartDate,
+                                     unit: unit,
+                                     staleGlucoseAge: staleGlucoseAge,
+                                     sensor: sensor)
         
-        let time = timeFormatter.string(from: glucoseStartDate)
-        caption?.text = time
+        glucoseValueHUD.glucoseLabel.text = viewModel.glucoseValueString
+        glucoseValueHUD.unitLabel.text = viewModel.unitsString
+        glucoseTrendHUD.setTrend(viewModel.trend)
+        tintColor = viewModel.tintColor ?? .label
+        accessibilityValue = viewModel.accessibilityString
         
-        isStaleAt = glucoseStartDate.addingTimeInterval(staleGlucoseAge)
```

Comment:
If the CGM goes offline; i.e. phone is out of range of the sensor, and the CGMManager does not report any data, we still need to show the user that the data is stale after a period of time. It appears that that behavior is being removed here.

Reply:
@ps2 This code was moved to the `CGMStatusHUDViewModel`. There is a timer that when fired would execute the block passed from the `CGMStatuHUDView`. Does this answer your question?

Reply:
Ah, ok. Thanks. Yes, that answers it. Will approve.